docs/support/coverage.md

```diff
@@ -6,18 +6,23 @@ To get the most out of Steamship's cloud platform you'll be using adapters. Stea
 ## Modules 
 
 * LLMs
-  * ✅ LLM Models
-  * ✅ LLM Caching 
-  * ❌ LLM Serialisation
-  * ✅ Token Usage Tracking 
-* Prompt Templates ✅
+  * ✅ LLM Models using `steamship_langchain.llms.OpenAI`
+  * ✅ LLM Caching using `steamship_langchain.cache.SteamshipCache`
+  * ❌ LLM Serialisation 
```

Comment:
curious: it seems like there is nothing preventing this, necessarily. what is left to be done?

Reply:
See Previous PR. There's a namespacing issue with our OpenAI implementation.